File: Loiane_Training_Java/Java_Basico/Exer Aula 18_19/Ex25.java
public class Ex25 {
    public static void main(String[] args) {
        int[] a = { 1, 3, 4, 6, 7, 9, 10, 12, 13, 15 };
        int[] b = new int[10];

        for (int i = 0; i < a.length; i++) {
            if (a[i] % 2 == 0) {
                b[i] = 1;
            } else {
                b[i] = 0;
            }
            System.out.println(b[i]);
        }
    }
}

File: loiane-training-csharp/procedural-poo/e/Ex25.cs
using System;

class Ex25
{
    static void Main(string[] args)
    {
        double[] vetorA = new double[10] { 1, 2, 3, 4, 5, 5, 4, 3, 2, 0 };
        double[] vetorB = new double[10];

        for (int i = 0; i < vetorA.Length; i++)
        {
            if (vetorA[i] % 2 == 0)
            {
                vetorB[i] = 1;
            }
            else
            {
                vetorB[i] = 0;
            }
        }

        for (int i = 0; i < vetorA.Length; i++)
        {
            Console.Write(vetorA[i] + " ");
        }

        System.Console.WriteLine();

        for (int i = 0; i < vetorA.Length; i++)
        {
            Console.Write(vetorB[i] + " ");
        }
    }
}